externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/HiveSQLEngine.scala

```diff
@@ -66,7 +65,6 @@ object HiveSQLEngine extends Logging {
   var currentEngine: Option[HiveSQLEngine] = None
   val hiveConf = new HiveConf()
   val kyuubiConf = new KyuubiConf()
-  kyuubiConf.set(ENGINE_EVENT_LOGGERS.key, "JSON")
```

Comment:
this is a behavior change.. the default value is `SPARK` and you just filter that

Reply:
one option maybe, we can add engine event loggers config for each engine.
e.g.
```
kyuubi.engine.spark.event.loggers
kyuubi.engine.jdbc.event.loggers
...
```

Reply:
cc @yaooqinn

Reply:
> one option maybe, we can add engine event loggers config for each engine.
> e.g.
> ```
> kyuubi.engine.spark.event.loggers
> kyuubi.engine.jdbc.event.loggers
> ...
> ```

good idea

Reply:
add config for each engine. https://github.com/apache/incubator-kyuubi/pull/3309#issue-1346621427